src/test/kotlin/org/pac4j/vertx/handler/impl/Functions.kt

```diff
@@ -65,7 +64,7 @@ fun logoutHandler(vertx: io.vertx.core.Vertx): Handler<RoutingContext> {
 }
 
 
-fun getProfileManager(rc: RoutingContext): VertxProfileManager<TestOAuth1Profile> {
```

Comment:
It's absolutely not a problem (for the tests): your choice. And it's always good to open up our minds (Ratpack tests are written in Groovy). But I hadn't notice before.